setup.py

```diff
@@ -30,6 +30,7 @@
         'pyyaml',
         'six',
         'webob',
+        'web-fragments',
```

Comment:
Agreed. I definitely won't merge this until web fragments 0.1.0 has been published.